rust/private/rust.bzl

```diff
@@ -714,8 +714,15 @@ _rust_test_attrs = {
     ),
 }
 
+_common_providers = [
```

Comment:
I believe this is not needed for this PR.

Reply:
I moved it out into another PR I forgot to open yesterday.